### ISSUE TITLE
keyboard_focus set to False is ignored if it is the first interactive element on the screen

### ISSUE THREAD

**Issue by TheSchnappi** [renpy/renpy#2550]
Hello,

it seems there is a bug with the keyboard_focus flag. If the button is the first interactive element on the screen, even if the keyboard focus should be ignored, it will be still focused.
This bug happens in 7.3.5 as well as in 7.4.0

Steps to recreate the bug:

1. Run the example code below in an empty test project
2. Press the down key and the button "Test 1 (no focus)" will wrongly be focused
3. Press the down key again, button "Test 3 (focus)" correctly be focused
4. Press the up key, now a button from the quick menu gets focused
5. Press the down key again and button "Test 1 (no focus)" gets the focus again

Example code:
```
define e = Character("Eileen")

screen focus_test():
    frame:
        vbox:
            textbutton "Test 1 (no focus)" keyboard_focus False action Return()
            textbutton "Test 2 (no focus)" keyboard_focus False action Return()
            textbutton "Test 3 (focus)" action Return()
            textbutton "Test 4 (no focus)" keyboard_focus False action Return()
            textbutton "Test 5 (no focus)" keyboard_focus False action Return()

# The game starts here.

label start:
    scene bg room
    show eileen happy

    show screen focus_test

    e "You've created a new Ren'Py game."
    e "Once you add a story, pictures, and music, you can release it to the world!"
    return
```

**Comment by midgethetree** [comment]
This is the issue fixed by #2616, it can be closed.
